test_scripts/LowVoltage/001_Low_Voltage_and_Wake_Up.lua

```diff
@@ -0,0 +1,253 @@
+---------------------------------------------------------------------------------------------------
+--
```

Comment:
@dboltovskyi Please add description of the script to header part of the script.

This comment is applicable to all scripts of current pull request.

Reply:
@aderiabin Accepted. Please check af2e393